utils/splash/Panoramas.js

```diff
@@ -10,22 +10,77 @@ export const PANORAMAS = [
         name: 'healthy',
         label: 'Healthy',
         image: 'https://s3.amazonaws.com/wri-api-backups/resourcewatch/staging/images/healthy-optimized.jpg',
-        markup: ''
+        intro:
+          <div>
+            <p>Airport Reef is located off the island of Tutuila, the largest island in American Samoa. It’s part of the National Marine Sanctuary of American Samoa, a NOAA protected area. In this photo, the staghorn corals are healthy and abundant. But researchers returned just two months later to find the reef completely transformed by severe coral bleaching. Around the world, coral reefs feed and shelter thousands of species of fish, buffer coastal areas against damaging waves, and support local economies through tourism and fishing.
+            </p>
+            <p><i>Photo date: December 1, 2014</i></p>
+          </div>,
+        hotspots: [
+          {
+            title: 'Coral reefs by the numbers',
+            id: 'coral_reefs_by_the_numbers',
+            position: '0 2 -12',
+            rotation: '0 0 0',
+            markup:
+              <div>
+                <h3>More than 4000</h3>
+                <p>fish species call <a href="http://www.wri.org/blog/2011/02/wake-call-save-coral-reefs" target="_blank" rel="noopener noreferrer">reefs home</a></p>
+                <h3>One quarter to one third</h3>
+                <p>of ocean species <a href="http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0025026" target="_blank" rel="noopener noreferrer">depend on tropical reefs.</a></p>
+                <h3>$29.8 billion</h3>
+                <p>is the <a href="http://www.icriforum.org/sites/default/files/Economic_values_global compilation.pdf" target="_blank" rel="noopener noreferrer">estimated yearly value</a> of all the services reefs provide, including tourism, coastal protection, fisheries and biodiversity</p>
+                <h3>500 million</h3>
+                <p>people <a href="http://www.noaa.gov/explainers/coral-reefs-essential-and-threatened" target="_blank" rel="noopener noreferrer">depend on coral reefs</a> for their food and livelihoods</p>
+                <h3>75</h3>
+                <p>percent of reefs worldwide <a href="http://www.reefresilience.org/pdf/Reefs_at_Risk_Revisited.pdf" target="_blank" rel="noopener noreferrer">are threatened</a> by warmer oceans and local threats such as coastal development, overfishing and pollution</p>
+              </div>,
+            image: '../../static/images/splash/coral_bleaching_on_the_rise.png',
```

Comment:
Do you really need to set this path as relative? I'm just asking if you should use this one `/static/images/splash/coral_bleaching_on_the_rise.png`

Reply:
I've just made a quick test and it seems to work as well if we use `/static/...` so I will update them.
I think we were using this relative path because we had problems in the past when referencing absolute ones with Aframe _(at least inside Aframe entities)_